test_script.py

```diff
@@ -155,17 +148,12 @@ def main():
     terminate_logging_event = threading.Event()
     test_failed_event = threading.Event()
 
-    nodes = [
-        Node('validatorA', 1, log_lines_queue, terminate_logging_event, test_failed_event),
-        Node('validatorB', 2, log_lines_queue, terminate_logging_event, test_failed_event),
-        Node('validatorC', 3, log_lines_queue, terminate_logging_event, test_failed_event),
-        Node('validatorD', 4, log_lines_queue, terminate_logging_event, test_failed_event),
-        Node('validatorE', 5, log_lines_queue, terminate_logging_event, test_failed_event),
-    ]
+    nodes = [Node('validator'+char, i+1, log_lines_queue, terminate_logging_event, test_failed_event)
+            for i, char in enumerate('ABCDE')]
```

Comment:
Might well be that in actuality you don't care for the name and

```python
nodes =  [Node('validator'+str(idx), idx,...) for idx in range(validator_nodes)]
```
is the way to go.